### PR TITLE
s3 parquet doc

### PR DESCRIPTION
**Pull request by y-f-u**

### CODE REVIEW

**Review by digadeesh — DISMISSED** [review]

**Review by ewgenius — DISMISSED** [review]

File: spiceaidocs/content/en/reference/Connectors/_index.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+type: docs
+title: "Connectors"
+linkTitle: "Connectors"
+description: ''
+weight: 70
+---
+
+## Connectors
```
Comment (lukekim) [inline spiceaidocs/content/en/reference/Connectors/_index.md]: `Connectors` -> `Data Connectors` in all places.

File: spiceaidocs/content/en/reference/Connectors/s3.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+---
+type: docs
+title: "S3 Parquet Dataset"
```
Comment (lukekim) [inline spiceaidocs/content/en/reference/Connectors/s3.md]: "S3 Parquet Dataset" -> `S3 Data Connector`

File: spiceaidocs/content/en/reference/Connectors/s3.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+linkTitle: "S3 Parquet Dataset"
+description: 'S3 Parquet Dataset YAML reference'
+weight: 80
+---
+
+Parquet data in S3-compatible storage can be queried in mesh mode or accelerated locally.
```
Comment (lukekim) [inline spiceaidocs/content/en/reference/Connectors/s3.md]: Use the language from the release notes.

File: spiceaidocs/content/en/reference/Connectors/s3.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+# `dataset`
+
+```yaml
+- from: s3://s3-bucket-name/path/to/parquet/file.parquet
+  name: s3_file
+  auth: s3
```
Comment (lukekim) [inline spiceaidocs/content/en/reference/Connectors/s3.md]: Should this be here?

File: spiceaidocs/content/en/reference/Connectors/s3.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+  params:
+    endpoint: https://your.minio.host
+    region: ""
+  acceleration:
+    enabled: true
+    refresh_mode: full
```
Comment (lukekim) [inline spiceaidocs/content/en/reference/Connectors/s3.md]: Do we technically need a `refresh_interval`

File: spiceaidocs/content/en/reference/Connectors/s3.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+```
+
+## `params`
+
+- `endpoint`: Check `with_endpoint` in [object store](https://docs.rs/object_store/latest/object_store/aws/struct.AmazonS3Builder.html)
```
Comment (lukekim) [inline spiceaidocs/content/en/reference/Connectors/s3.md]: This is confusing, we should describe what they are and provide examples.

File: spiceaidocs/content/en/reference/Connectors/s3.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+- `region`: Check `with_region` in [object store](https://docs.rs/object_store/latest/object_store/aws/struct.AmazonS3Builder.html)
```
Comment (lukekim) [inline spiceaidocs/content/en/reference/Connectors/s3.md]: Provide example.

File: spiceaidocs/content/en/reference/Connectors/s3.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+
+## `acceleration`
```
Comment (lukekim) [inline spiceaidocs/content/en/reference/Connectors/s3.md]: Acceleration is not specific to S3, we should remove this section (or reference general acceleration doc).

File: spiceaidocs/content/en/reference/Connectors/s3.md [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+
+Optional.
+
+When `null`, query will run in mesh mode; parquet data will be read directly from the object store for each query.
+
+Only `full` refresh mode is supported.
+
+## `auth`
+
+Check [Secrets]({{<ref "reference/Secrets">}}).
```
Comment (lukekim) [inline spiceaidocs/content/en/reference/Connectors/s3.md:18]: This looks wrong? But also need example.

File: spiceaidocs/content/en/reference/Connectors/s3.md (hunk 4 above)
Comment (ewgenius) [inline spiceaidocs/content/en/reference/Connectors/s3.md]: We don't have `auth` field in dataset, is that actually used?

Comment (ewgenius) [inline spiceaidocs/content/en/reference/Connectors/s3.md]:
```suggestion
```

File: spiceaidocs/content/en/reference/Connectors/_index.md (hunk 1 above)
Comment (ewgenius) [reply spiceaidocs/content/en/reference/Connectors/_index.md]: title not needed here, we have it from template
CleanShot 2024-03-15 at 11 04 49@2x

**Review by lukekim — APPROVED** [review]

**Pull request merged**